### PR TITLE
Corrected typo

### PR DESCRIPTION
**Pull request by eedrxs**
### Description

 Corrected a typo.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks, but this is not a typo or grammatically incorrect.

**Comment by eedrxs** [comment]
@hamishwillee This is very much a grammatical error, chief.

**Comment by hamishwillee** [comment]
OK, technically it is, but it's in such common use there's no ambiguity or likely confusion. Thanks for your contribution.

**Review by hamishwillee — APPROVED** [review]

File: files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ This is the difference between one and two-dimensional layout. In a one dimensio
 
 Typically flexbox-based grid systems work by taking flexbox back to the familiar world of float-based layouts. If you assign percentage widths to flex items — either as `flex-basis` or by adding a width to the item itself leaving the value of `flex-basis` as `auto` — you can get the impression of a two dimensional layout. You can see this working in the example below.
 
-Here I have set `flex-grow` and `flex-shrink` to `0` to make inflexible flex items and am then controlling flexibility using percentages, just like we used to do in float layouts.
+Here I have set `flex-grow` and `flex-shrink` to `0` to make inflexible flex items and I'm then controlling flexibility using percentages, just like we used to do in float layouts.
```
Comment (Josh-Cena) [inline files/en-us/web/css/css_flexible_box_layout/mastering_wrapping_of_flex_items/index.md:54]: FWIW, if we are using "and" to join to complete sentences, it's more correct to say:

```suggestion
Here I have set `flex-grow` and `flex-shrink` to `0` to make inflexible flex items, and I'm then controlling flexibility using percentages, just like we used to do in float layouts.
```

But it definitely sounds better than "and am then".

**Pull request merged**